sqlserver/check.py

```diff
@@ -636,7 +636,7 @@ def __init__(self, connector,  cfg_instance, base_name,
         self.connector = connector
         self.cfg_instance = cfg_instance
         self.datadog_name = cfg_instance['name']
-        self.sql_name = cfg_instance['counter_name']
+        self.sql_name = cfg_instance.get('counter_name', '')
```

Comment:
Does this make more sense than `sql_name` being `None`? I see some queries get executed with `self.sql_name` as an argument, so I guess this is what we want...